cmake/FindKokkos.cmake

```diff
@@ -44,6 +44,10 @@ The following cache variables may also be set:
 
 find_package(PkgConfig)
 pkg_check_modules(PC_Kokkos QUIET kokkos)
+if(NOT PC_Kokkos_FOUND)
+  message(FATAL_ERROR "The PkgConfig module for Kokkos was not found. "
+    "Please make sure that PKG_CONFIG_PATH includes the directory containing kokkos.pc.")
+endif()
```

Comment:
Original intent was to use it only as a hint but I think what you propose is the right thing to do